l5kit/l5kit/evaluation/metrics.py

```diff
@@ -1,23 +1,18 @@
 import numpy as np
 
 
-def neg_multi_log_likelihood(
-    gt: np.ndarray, pred: np.ndarray, confidences: np.ndarray, avails: np.ndarray
-) -> np.ndarray:
+def _assert_shapes(gt: np.ndarray, pred: np.ndarray, confidences: np.ndarray, avails: np.ndarray) -> None:
     """
-    Compute a negative log-likelihood for the multi-modal scenario.
-    log-sum-exp trick is used here to avoid underflow and overflow, For more information about it see:
-    https://en.wikipedia.org/wiki/LogSumExp#log-sum-exp_trick_for_log-domain_calculations
-    https://timvieira.github.io/blog/post/2014/02/11/exp-normalize-trick/
-    https://leimao.github.io/blog/LogSumExp/
+    Check the shapes of args required by metrics
 
     Args:
         gt (np.ndarray): array of shape (time)x(2D coords)
```

Comment:
Maybe replacing 'time' with 'future timesteps' or something like that is a bit more usual?